west.yml

```diff
@@ -39,7 +41,7 @@ manifest:
   defaults:
     remote: ncs
 
-  group-filter: [-homekit, -nrf-802154, -find-my, -ant]
+  group-filter: [-homekit, -nrf-802154, -dragoon-src, -find-my, -ant]
```

Comment:
Please rename `dragoon-src` to `dragoon`, we don't want to add qualifiers to group filters, just names

Reply:
Done